hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/repository/CryptoAllowanceHistoryRepository.java

```diff
@@ -27,6 +27,6 @@
 
     @Modifying
     @Override
-    @Query(nativeQuery = true, value = "delete from crypto_allowance_history where upper(timestamp_range) <= ?1")
+    @Query(nativeQuery = true, value = "delete from crypto_allowance_history where timestamp_range << int8range (?1, null)")
```

Comment:
nit, remove the extra space

```suggestion
    @Query(nativeQuery = true, value = "delete from crypto_allowance_history where timestamp_range << int8range(?1, null)")
```

Reply:
Done (for all 5 Java files).